### PR TITLE
add link in logo

### PR DESCRIPTION
**Pull request by graykode**

### CODE REVIEW

File: components/header.tsx [hunk 1]
```diff
@@ -12,8 +12,10 @@ const Header = ({ onMenuClick }: { onMenuClick: () => void }) => {
   return (
     <div className="fixed w-full flex justify-between items-center py-4 px-4 sm:px-8 bg-white bg-opacity-5 backdrop-blur">
       <div className="flex items-center gap-12">
-        <LogoSvg className="h-9 hidden sm:flex w-fit" />
-        <LogotypeSvg className="h-4 sm:hidden w-fit" />
+        <a target="_blank" href="/" rel="noreferrer">
```
Comment (onemonster) [inline components/header.tsx:15]: We should use the `<Link>` component for next.js

Comment (onemonster) [reply components/header.tsx:15]: https://nextjs.org/docs/pages/api-reference/components/link

Comment (graykode) [reply components/header.tsx]: https://github.com/clober-dex/coupon.finance/pull/36/commits/f6ba5c0757dd5bf0edf1cc2a14caa0d960cd571a

**Review by onemonster — APPROVED** [review]
LGTM

**Pull request merged**